test/integration_test/basic_test.go

```diff
@@ -989,6 +997,33 @@ func BasicSecurityRegression(tc *types.TestCase) func(*testing.T) {
 	}
 }
 
+// BasicKvdbRegression test includes the following steps:
+// 1. Deploy PX and validate internal KVDB
+// 2. Validate KVDB pods and other components
+// 3. Delete KVDB pods and validate they get redeployed
+// 4. Delete StorageCluster and validate it got successfully removed
+func BasicKvdbRegression(tc *types.TestCase) func(*testing.T) {
+	return func(t *testing.T) {
+		var err error
+		testSpec := tc.TestSpec(t)
+		cluster, ok := testSpec.(*corev1.StorageCluster)
+		require.True(t, ok)
+
+		// Create and validate StorageCluster
+		cluster = ci_utils.DeployAndValidateStorageCluster(cluster, ci_utils.PxSpecImages, t)
+
+		// Delete all KVDB pods and validate the get re-created
+		logrus.Info("Delete portworx KVDB pods and validate they get re-deployed")
+		err = coreops.Instance().DeletePodsByLabels(cluster.Namespace, map[string]string{"kvdb": "true"}, 120*time.Second)
```

Comment:
are we able to delete all kvdb pods at once?

Reply:
Yes, those are just pods and have nothing to do with an actual KVDB service